### PR TITLE
Feature/Insert mock data [#20]

### LINKED ISSUE #20: Add mock data to db for testing the endpoints and SQL queries
**Is your feature request related to a problem? Please describe.**
The mock data will be used for implementing the endpoints and testing the SQL queries that retrieve data from db.

**Describe the solution you'd like**
Add mock (coherent) data for the tables **Ingredient**, **CoffeeRecipe** and **CoffeeRecipeIngredient** respectively.

### PR DESCRIPTION
**Pull request by alcxanndra**
## Description
This PR adds data.sql file for mock data insert stataments and adopts uppercase convention for SQL data types to better differentiate from column names that are lowercased.

## Issue
Closes #20

### CODE REVIEW

**Review by vladpirlog — COMMENTED** [review]
does sqlite accept backticks? i noticed there are no backticks in `schema.sql`

**Comment by alcxanndra** [comment]
> does sqlite accept backticks? i noticed there are no backticks in `schema.sql`

Backticks are accepted (for consistency with my SQL) though not necessary. Removed them for consistency.

**Review by vladpirlog — APPROVED** [review]

**Comment by vladpirlog** [comment]
please resolve the conflicts, squash and rebase

File: schema.sql [hunk 1]
```diff
@@ -1,36 +1,39 @@
 CREATE TABLE Ingredient (
-  id integer PRIMARY KEY AUTOINCREMENT,
-  name varchar(255) NOT NULL,
-  measure_unit varchar(255) NOT NULL,
-  available DECIMAL(10,5) NOT NULL
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
```
Comment (vladpirlog) [inline schema.sql:2]: should be `AUTOINCREMENT` instead of `AUTO_INCREMENT` (see [here](https://sqlite.org/autoinc.html))

File: schema.sql [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
+  name TEXT NOT NULL,
+  measure_unit TEXT NOT NULL,
+  available REAL NOT NULL
 );
 
 CREATE TABLE CoffeeRecipe (
-  id integer PRIMARY KEY AUTOINCREMENT,
-  name varchar(255) NOT NULL
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
```
Comment (vladpirlog) [inline schema.sql:9]: same

File: schema.sql [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
-  FOREIGN KEY (recipe_id) REFERENCES CoffeeRecipe (id)
+  ingredients_json TEXT NOT NULL,
+  PRIMARY KEY (recipe_id, started_at)
 );
 
 CREATE TABLE Log (
-  id integer PRIMARY KEY AUTOINCREMENT,
-  type varchar(255) NOT NULL,
-  properties_json varchar(255) NOT NULL,
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
```
Comment (vladpirlog) [inline schema.sql:29]: same

**Review by vladpirlog — CHANGES_REQUESTED** [review]

**Review by vladpirlog — APPROVED** [review]

**Pull request merged**